app/release_metadata.json

```diff
@@ -1,11 +1,11 @@
 {
-  "version": "4.4.7",
+  "version": "4.4.8",
   "stability": "stable",
   "minimum_php_version": "7.4.0",
   "maximum_php_version": "8.0.99",
   "minimum_mysql_version": "5.7.14",
   "minimum_mariadb_version": "10.2.7",
   "show_php_version_warning_if_under": "7.4.0",
   "minimum_mautic_version": "3.2.0",
-  "announcement_url": "https://github.com/mautic/mautic/releases/tag/4.4.7"
+  "announcement_url": "https://github.com/mautic/mautic/releases/tag/4.4.8"
```

Comment:
@IonutOjicaDE this file should not be edited with your PR, please reset this file to what we have in branch `4.4` an easy way to do it via

```
git checkout 4.4 -- app/release_metadata.json
```

Reply:
@mabumusa1 I did not changed this file (release_metadata.json). I assume, I chosen the branch 4.4.8 and I should have picked the 4.4.
I installed git and cloned my repository. But the command is give me error (fatal: invalid reference: 4.4):
![image](https://github.com/mautic/mautic/assets/96743055/60ebad16-3b2c-4f5c-aa33-2179a4708ace)
I do not know further.
PS: the changes of the library file I made them directly in github, not by using git.